src/node_api_internal.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SRC_NODE_API_INTERNAL_H_
+#define SRC_NODE_API_INTERNAL_H_
+
+#include "node_api.h"
+#include "node.h"
+
+namespace v8impl {
+  napi_env JsEnvFromV8Isolate(v8::Isolate* isolate);
+  v8::Isolate* V8IsolateFromJsEnv(napi_env e);
+
+  napi_value JsValueFromV8LocalValue(v8::Local<v8::Value> local);
+  v8::Local<v8::Value> V8LocalValueFromJsValue(napi_value v);
```

Comment:
Resolved: There was no good reason for the `_internal.h` headers anymore, so I removed them in https://github.com/nodejs/node/pull/11975/commits/f21ab80825e62b302da65abbea49cff834650668